tests/periodic_box_freesurface/screen-output

```diff
@@ -40,48 +40,48 @@ Number of free surface degrees of freedom: 3246
      reference_cp:                                  1250
      reference_thermal_diffusivity:                 1.13939e-06
 
-     RMS, max velocity:  0.626 m/year, 1.23 m/year
+     RMS, max velocity:  0.646 m/year, 8.25 m/year
```

Comment:
uh, that's a large difference in the max velocity. can you visually inspect whether everything still looks alright? the average did not change very much, so it looks like a singularity somewhere.